test/ffmpeg-qsv/encode/encoder.py

```diff
@@ -52,6 +52,10 @@ def gen_output_opts(self):
       opts += " -refs {refs}"
     if vars(self).get("lowpower", None) is not None:
       opts += " -low_power {lowpower}"
+    if vars(self).get("lookahead", None) is not None:
+      opts += " -look_ahead {lookahead}"
+    if vars(self).get("lookaheaddepth", None) is not None:
+      opts += " -look_ahead_depth {lookaheaddepth}"
```

Comment:
These pipeline opts should be defined together based on `la_depth` presence.  That is, you can't have one without the other.